### ISSUE TITLE
Can't use uncompressed Blockly from the filesystem after updating the closure library

### ISSUE THREAD

**Issue by rachel-fenichel** [google/blockly#1404]
TLDR: the workaround is to run a server and work from localhost.  
[http-server](https://www.npmjs.com/package/http-server) and [simplehttpserver](http://www.pythonforbeginners.com/modules-in-python/how-to-use-simplehttpserver/) are both fast to set up.  
Make sure your server has access to both blockly and closure-library.

### Problem statement
- [x] Bug report
- [ ] Feature request

Problem loading the playground (or any file that uses blockly_uncompressed.js) from the file system after updating closure.

### Expected Behavior

Blockly loads

### Actual Behavior

Blockly does not load.  There is an error about cross-origin requests.

### Steps to Reproduce

Update closure-library.
Rebuild
Open the playground
### Stack Traces

```

base.js:1924 Failed to load file:///home/fenichel/blockly/closure-library/closure/goog/deps.js: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, https.
goog.loadFileSync_ @ base.js:1924
goog.retrieveAndEval_ @ base.js:918
goog.getLoader_ @ base.js:1762
goog.addDependency @ base.js:616
window.BLOCKLY_BOOT @ blockly_uncompressed.js:41
(anonymous) @ playground.html:6
```

### Operating System and Browser

- [ ] Desktop:
  - [x] Chrome
  - [ ] Firefox
  - [ ] Safari
  - [ ] Opera
  - [ ] IE 10+
  - [ ] IE 11
  - [ ] EDGE

- [ ] Smartphone/Tablet/Chromebook (please complete the following information):

**Comment by rachel-fenichel** [comment]
Whatever broke us is a change since October 12 on closure: https://github.com/google/closure-library/commits/master

**Comment by rachel-fenichel** [comment]
First failure is here: https://github.com/google/closure-library/commit/c8cb1cc094c2a469e5151bb01dc8d218d293aa7e

But on that one it's actually failing to load closure, which is a different error.  
```
base.js:925 Uncaught SyntaxError: Unexpected token }
```

The commit that followed immediately also fails with "Unexpected token": https://github.com/google/closure-library/commit/aa92ba3c3cf3861baec980ae2c05d9ae265d9841

The next commit is the first one with the failure shown above.
https://github.com/google/closure-library/commit/8086789e7883d0a96b1a0912fdb94bb7f1b1357f

**Comment by NeilFraser** [comment]
Confirmed.  The last version of Closure that works is [c8cb1cc094c2a469e5151bb01dc8d218d293aa7e](https://github.com/google/closure-library/commit/c8cb1cc094c2a469e5151bb01dc8d218d293aa7e).

The [next version](https://github.com/google/closure-library/commit/aa92ba3c3cf3861baec980ae2c05d9ae265d9841) is the one that introduces the bug.  This version also introduces a syntax error (!) that's fixed in the [version after that](https://github.com/google/closure-library/commit/8086789e7883d0a96b1a0912fdb94bb7f1b1357f).

**Comment by NeilFraser** [comment]
This is not just a Blockly issue.  Closure is no longer viable on file URLs.  Closure's own "[hello world](https://developers.google.com/closure/library/docs/gettingstarted)" demo fails.

Fixing this in Blockly (assuming it is possible) does not appear to be the correct course of action at this time.  Suggest leaving Blockly alone for now.  I've [filed a bug](https://github.com/google/closure-library/issues/851) with Closure.

**Comment by NeilFraser** [comment]
Tests indicate that this issue has been successfully resolved by the Closure Library team.  Woo!
